doc/fr/upgrade/from_packages_34.rst

```diff
@@ -81,12 +86,30 @@ suivante : ::
 .. note::
     Changez **Europe/Paris** par votre fuseau horaire.
 
-Redémarrez les services en Executer les commandes suivantes : ::
+Réalisez les actions suivantes : ::
 
+    # systemctl disable rh-php71-php-fpm
+    # systemctl stop rh-php71-php-fpm
+    # systemctl stop rh-php72-php-fpm
     # systemctl enable rh-php72-php-fpm
```

Comment:
start before enable ?

Reply:
Order doesn't matter, _start_ and _enable_ do different things.